src/styles/blocks/nav.scss

```
@@ -0,0 +1,166 @@
+.nav {
+  @media (max-width: 1236px) {
+    margin: 0 auto;
```

```suggestion
    margin: 0 auto;
```
Don't use outer spacings or blocks. You can use mix `nav header__nav` and give margins to `header__nav`